.github/workflows/release.yml

```diff
@@ -7,7 +7,6 @@ on:
 
 jobs:
   release_on_push:
-    if: github.event_name == 'pull_request' && github.event.action == 'closed' && github.event.pull_request.merged == true
```

Comment:
I copy/pasted this from the readme of a different action I almost used instead of the one I went with.